packages/react/src/hooks/utils/parseContractResult.test.ts

```diff
@@ -0,0 +1,47 @@
+import { BigNumber } from 'ethers'
+
+import { parseContractResult } from './parseContractResult'
+
+const contractInterface = [
+  {
+    inputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256',
+      },
+    ],
+    name: 'gms',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'timestamp',
+        type: 'uint256',
+      },
+      {
+        internalType: 'address',
+        name: 'sender',
+        type: 'address',
+      },
+    ],
+    stateMutability: 'view',
+    type: 'function',
+  },
+]
+
+describe('parseContractResult', () => {
+  it('should parse the data to an ethers Result if there are no named keys', () => {
```

Comment:
Can we add one more test for a plan result that skips this logic? Could use mloot or wagmigotchi.